### PR TITLE
updateing credits and CHANGELOG

### PR DESCRIPTION
**Pull request by tobiasd**
recreation of friendica/#2801 as with the original

this shall be updated in case there are updates before the release. So keep it open till then.

### CODE REVIEW

**Comment by rabuzarus** [comment]
note: I think we should make https://github.com/friendica/friendica/pull/2854 a little bit more prominent because it fixes https://github.com/friendica/friendica/issues/2835 which was a really important bug

**Comment by Hypolite** [comment]
Unfortunately, #2835 isn't fixed. 😞

File: CHANGELOG [hunk 1]
```diff
@@ -17,26 +17,37 @@ Version 3.5.1
...
   Friendica Addons:
-    Updates to the translations (DE, ES, FR, PT-BR) [translation teams]
+    Updates to the translations (DE, ES, FR, IT, PT-BR) [translation teams]
     Improvements to the IFTTT addon [Hypolite]
     Improvements to the language filter addon [strk]
     Improvements to the pump.io bridge [heluecht]
+    Improvements to the jappixmini addon [heluecht]
+    Improvements to the gpluspost addon [heluecht]
+    Improvements to the performance of the Twitter bridge when using workers [heluecht]
+    Diaspora Export addon is not working again [heluecht]
```
Comment (annando) [inline CHANGELOG]: "now", not "not" :-)

Comment (Quix0r) [inline CHANGELOG]: Oh, yeah. That is what I missed.

**Comment by Quix0r** [comment]
Busy reviewer ... ;-)

**Pull request merged**